docs/getting_started/installation_en.md

```diff
@@ -0,0 +1,100 @@
+# Installation
+
+### How to install from pypi
```

Comment:
Most issues are not coming from pip install, they show up when user launch visualdl, so it should include running the demo script and launch visualdl to verify installation is good.

I think "Install with native pip" would be a better title, coz either using Virtualenv or Anaconda or none of them, we are also still installing from pypi

I think just copied the "Quick Start" section from what we had and change title to "Install with native pip" should be good as following:

## Install with native pip

```
# Install the VisualDL. Preferably under a virtual environment or anaconda.
pip install --upgrade visualdl

# run a demo, vdl_create_scratch_log will create logs for testing.
vdl_create_scratch_log
visualdl --logdir=scratch_log --port=8080

# visit http://127.0.0.1:8080
```